conf.d/main

```diff
@@ -5,18 +5,19 @@ DB_NAME=yii2
 DB_USER=yii2
 DB_PASS=$(mcookie)
 
-SRC=/usr/local/src
 WEBROOT=/var/www/yiiframework
 
 # unpack
-cd $SRC
-tar -zxf yii-*.tgz -C .
 mkdir -p $WEBROOT
-mv $SRC/advanced/* $WEBROOT
-rm $SRC/yii-*.tgz
 
 cd $WEBROOT
 
+composer global require "fxp/composer-asset-plugin:^1.2.0"
+composer create-project yiisoft/yii2-app-advanced advanced 2.0.11
+
+mv advanced/* .
+rm -rf advanced
```

Comment:
Happy days all round... PR merged. I'll build this today ready for testing. Assuming all goes well should be published next week.

Also @cebe would you mind dropping me a line at your convenience. Get me on jeremy@turnkeylinux.org I just like to share some stuff on our plans moving forward. It's not secret, but I'd prefer something not quite so public for now.